scripts/bench/openfdd_bacnet_poll_daemon.sh

```
@@ -0,0 +1,146 @@
+    fdd_ok=false
+
+    token="$(openfdd_auth_login_token "$BASE" "$AUTH" integrator 2>/dev/null || true)"
+    if [[ -n "$token" ]]; then
+      local hdr=(-H "Authorization: Bearer $token")
+      local live modbus_read_ok=false bacnet_read_ok=false modbus_value="" bacnet_value=""
+      jq -e '.ok == true' <<< "$(curl "${CURL_TLS[@]}" -fsS "${hdr[@]}" "$BASE/api/health" 2>/dev/null || echo '{}')" \
+        >/dev/null 2>&1 && stack_ok=true
+
+      live="$(openfdd_bench_live_ot_poll "$BASE" "$COMMISSION" "$token" "$ROOT")"
+      echo "$live" >"$OUT/live_ot_cycle_${n}.json"
+      jq -e '.modbus_read_ok == true' <<<"$live" >/dev/null 2>&1 && { modbus_read_ok=true; modbus_ok=true; }
```

Shellcheck (0.11.0)

[warning] modbus_ok appears unused. Verify use (or export if used externally).

(SC2034)